woocommerce/woocommerce-ios
# Custom help content for "Store picker" screen

Part of: #7547 

### Description
As a part of improving the help center content, this PR adds custom content with FAQs and Answers for the "Store picker" screen at the following URL.

https://woocommerce.com/document/android-ios-apps-login-help-faq/#pick-store-after-entering-password

Internal ref - pe5sF9-sZ-p2

### Testing instructions

**You will need to have more than one WooCommerce store connected with your WordPress.com account. This is needed in order to view store picker screen.** 

1. Install and launch the app.
1. Tap `Skip` to reach the login prologue screen.
1. Tap the `Continue with WordPress.com` button.
1. Login using your WordPress.com account that has multiple WooCommerce stores.
1. You will land in the "Store picker" screen from which you can choose a WooCommerce store to proceed into the app.
1. Tap `Help` -> `Help Center`
1. Observe that you are directed to https://woocommerce.com/document/android-ios-apps-login-help-faq/#pick-store-after-entering-password in a web view
1. In Xcode debug console observe that the following event is tracked. Note that the `source_flow` property will differ based on the login method you selected. 
```
Tracked support_help_center_viewed, properties: [AnyHashable("help_content_url"): "https://woocommerce.com/document/android-ios-apps-login-help-faq/#pick-store-after-entering-password", AnyHashable("source_flow"): "login_password_magic_link_emphasis", AnyHashable("source_step"): "site_list"]
```

### Screenshots

<img src="https://user-images.githubusercontent.com/524475/188127601-d7c4755f-2fee-4d36-9c38-b55c6b2fc35f.png" width="300" />

---
